test/models/work_test.rb

```diff
@@ -0,0 +1,53 @@
+require "test_helper"
+
+describe Work do
+  describe 'validations' do
+    before do
+      # Arrange
+      @work = Work.new(title: 'test')
+    end
+
+    it 'is invalid without a title' do
+    # Arrange
+    @work.title = nil
+
+    # Act
+    result = @work.valid?
+
+    # Assert
+    expect(result).must_equal false
+    expect(@work.errors.messages).must_include :title
+    end
+  end
+  describe "sort_works" do
+    it "sorts works based on votes" do
+      sorted = Work.sort_works("album")
+      first = sorted.first.votes.count
```

Comment:
This is a good start to testing for your custom model methods, but there's some interesting edge cases you haven't covered.

For `find_top_work`, I would wonder:

- What happens if there are no works?
- What happens if there are works but no votes?
- What happens if two works have the same number of votes?

Similarly for your `sort_works` and `top_ten` methods, I would ask:

- What if there are no works of that category?
- What if there are less than 10 works?
- What if there's a tie for last place, e.g. works 9, 10 and 11 all have 0 votes?